workshop/controllers/ui/components/autoSave.lua

```diff
@@ -0,0 +1,17 @@
+-- Auto Save 
+local shared = require("tevgit:workshop/controllers/shared.lua")
+local autoSave = {}
+
+autoSave.Enabled = false
+autoSave.timeLimit = 5 -- Default (move to settings value soon)
+
+autoSave.Sync = spawnThread(function()
```

Comment:
once this thread is killed, autosave is killed and cannot be toggled back on

Reply:
Meant to wrap this in a ``function() end)``. That's on me.

Reply:
Will be fixed with the next commit to this pull.